src/navigation/actions.js

```diff
@@ -266,37 +266,40 @@ export const gotoStocktakes = () =>
  * Pushes the Stock route onto the main navigation stack.
  */
 export const gotoStock = () =>
-  NavigationActions.navigate({
-    routeName: ROUTES.STOCK,
+  CommonActions.navigate({
+    name: ROUTES.STOCK,
     params: {
       title: navStrings.current_stock,
     },
   });
 
 /**
- * Action creator for navigating to the SupplierRequisition screen.
+ * Action creator for navigating to the Stocktake Manage screen.
  *
- * @param {Object} requisition The requisition to pass to the next screen.
+ * @param {String} stocktakeName The name of the stocktake to pass to the next screen.
+ * @param {Object} stocktake The stocktake to pass to the next screen.
  */
 export const gotoStocktakeManagePage = (stocktakeName, stocktake) => dispatch => {
-  const currentRouteName = RootNavigator.getCurrentRouteName();
+  const currentName = RootNavigator.getCurrentRouteName();
```

Comment:
I think `currentRouteName` has a little more meaning and would be good to retain

Reply:
I agree, I found it a bit odd `react-navigation` moved from calling it `routeName` to just `name` in their API but good to keep the context where we can 😸  cb91d854